### ISSUE TITLE
Tablet & other touch device (phone) support?

### ISSUE THREAD

**Issue by garrett** [cthiel/cardboard#11]
It might be nice. Not currently on the radar, just noted for future work.

It would probably be fantastic for meetings. (Or we can ensure that the software just works _very well_ with projectors.)

**Comment by cthiel** [comment]
:+1:

**Comment by garrett** [comment]
I did look into this and started making it work on my iPad. However, it was view-only and a little buggy. It mostly works as-is now, though... but I'd like to make it better.

It is nifty to see the stories update after changing it on the laptop, though. (:

**Comment by cthiel** [comment]
Does dragging work for you? It didn't for me on my iPhone :)

**Comment by garrett** [comment]
No, that's what I meant by "view-only" for the iPad. I need to fix it somehow.

**Comment by bear454** [comment]
Drag doesn't seem to work on my Android devices either.  It looks like the ul is getting focus of the press.  No auto-refresh either :/

**Comment by garrett** [comment]
Auto-refresh works on my iPad. The touch events do not, however. If I fix it for one, I'll probably fix it for all. I haven't checked it out on my Android phone yet.

**Comment by garrett** [comment]
This should include @cthiel's suggestion of adding icons for iOS, so it displays nicely on the home screen.

**Comment by garrett** [comment]
We could switch everything to "touch" support and use this library, and things would be consistent:
https://github.com/davidcalhoun/touche

(Alternatively, we could simply bind functions to multiple events.)
